versions/2016-10-31.md

```diff
@@ -233,7 +237,7 @@ AccessLogSetting | [CloudFormation AccessLogSetting property](https://docs.aws.a
 CanarySetting | [CloudFormation CanarySetting property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigateway-stage-canarysetting.html) | Configure a Canary Setting to a Stage of a regular deployment. This value is passed through to Cloudformation, so any value supported by `CanarySetting` is supported here.
 TracingEnabled | `boolean` | Indicates whether active tracing with X-Ray is enabled for the stage.
 Models | `List of JSON or YAML objects` | JSON schemas that describes the models to be used by API methods.
-
+Domain | `Object` | Configuration settings for custom domains on API. Must contain `DomainName` and `CertificateArn`
```

Comment:
We usually don't just say `Object` for the type when it's an object we're defining. Can you do something similar to the `Cors` property and define a `Domain Configuration` object so you can specify the parameters on that object in the spec?